app/commands/api/v1/reports/fetch_keyword.rb

```diff
@@ -0,0 +1,187 @@
+module Api
+  module V1
+    module Reports
+      class FetchKeyword
+        prepend SimpleCommand
+        prepend SimpleResponse
+
+        def initialize(keyword_id)
+          @keyword_id = keyword_id
+        end
+
+        def call
+          return unless keyword
+          print("========>keyword.value : ", keyword.value)
```

Comment:
These debugging statements should be removed after the implementation is finalized